thunder-tests/thunderclient.json

```diff
@@ -1371,10 +1412,186 @@
       "files": [
         {
           "name": "files",
-          "value": "../../../../../Downloads/2022-07-25_C53_LI51088110104693K000U_USD_082598.xml"
+          "value": "kyc-incorp-cert/test.json"
         }
       ]
     },
     "tests": []
+  },
+  {
+    "_id": "848e8f65-ea05-4e9c-bbb2-4fef0ea60a27",
+    "colId": "62eefab7-84cd-496b-8f93-253c15d4f841",
+    "containerId": "37c18902-ffd3-446a-acc7-461024d278b8",
+    "name": "Request KYC",
+    "url": "{{url}}/v1/userData/{{userDataId}}/kyc",
+    "method": "PUT",
+    "sortNum": 510000,
+    "created": "2022-07-21T11:42:55.550Z",
+    "modified": "2022-07-21T11:44:36.177Z",
+    "headers": [],
+    "params": [],
+    "auth": {
+      "type": "bearer",
+      "bearer": "{{token}}"
+    },
```

Comment:
Can be removed (for every call). Just use inherited auth.

Reply:
How to use inherited Auth? If I go to None, there is a checkbox to disable it, but even if not disabled and it stating "Inherit Authentication values from Parent" it is not working.

Reply:
Well, it is working for all other requests 😁 try reloading Thunder or restarting VS Code. Might be some caching issue.

Reply:
layer 8 problem. thought i had still a valid token, but it wasn't. with a new one it is working.